<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationForm</class>
 <widget class="QWidget" name="CalibrationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Image selection</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="image_lw"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="open_files_btn">
          <property name="text">
           <string>Open Files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="open_dir_btn">
          <property name="text">
           <string>Open Dir</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="delete_btn">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clear_btn">
          <property name="text">
           <string>Clear All</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Output selection</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="output_file_le">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="select_btn">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="general_param_group_box">
     <property name="title">
      <string>General parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QComboBox" name="modality_cbb">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>astigmatism</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>arbitary</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2D PSF</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>3D modality</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Correct bead Z-positions using</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="filter_size_le">
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Filter size for peak finding</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Distance between frames(nm):</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Minimum distance (pixels)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="min_distance_le">
        <property name="text">
         <string>20</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="distance_btw_frames_le">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QComboBox" name="correct_bead_z_cbb">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>none</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>cross-correlation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>shape (astig)</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="frames_lbl">
          <property name="text">
           <string>frames to use for CC:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="frames_le">
          <property name="text">
           <string>50</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="frames_lbl_2">
          <property name="text">
           <string>relative cutoff</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="relative_cutoff_le">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Cspline parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>ROI size: X,Y (pixels)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="roi_size_xy_le">
        <property name="text">
         <string>27</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Smoothing parameters in Z</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="smoothing_param_le">
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gauss_fit_param_gb">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Gauss fit parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>ROI size (pixels)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Range (nm):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="gauss_fit_roi_size_le">
        <property name="text">
         <string>19</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>minimum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="gauss_fit_range_min_le">
          <property name="text">
           <string>-700</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>maximum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="gauss_fit_range_max_le">
          <property name="text">
           <string>700</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="use_zernike_fit_cb">
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Use Zernike fitting</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="zernike_fit_gb">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Zernike fit parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="lambda_l">
          <property name="text">
           <string>lambda:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lambda_le">
          <property name="text">
           <string>680</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pixelsizex_l">
          <property name="text">
           <string>pixel size X</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pixelsizex_le">
          <property name="text">
           <string>110</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="pixelsizey_l">
          <property name="text">
           <string>pixel size Y</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="pixelsizey_le">
          <property name="text">
           <string>110</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="otfrescale_l">
          <property name="text">
           <string>otf rescale (pixels)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="otfrescale_le">
          <property name="text">
           <string>0.5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="iterations_l">
          <property name="text">
           <string>iterations</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="iterations_le">
          <property name="text">
           <string>75</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="na_l">
          <property name="text">
           <string>NA</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="na_le">
          <property name="text">
           <string>1.5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="refmed_l">
          <property name="text">
           <string>Refractive index: medium</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="refmed_le">
          <property name="text">
           <string>1.518</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="refcov_l">
          <property name="text">
           <string>coverslip:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="refcov_le">
          <property name="text">
           <string>1.524</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="refimm_l">
          <property name="text">
           <string>immersion oil:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="refimm_le">
          <property name="text">
           <string>1.518</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="calibrate_btn">
     <property name="text">
      <string>Calculate beads calibration</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="status_lbl">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
